deepinv/loss/regularisers.py

```diff
@@ -18,6 +18,7 @@ class JacobianSpectralNorm(Loss):
     :param float tol: tolerance for the convergence of the power method.
     :param bool eval_mode: set to `False` if one does not want to backpropagate through the spectral norm (default), set to `True` otherwise.
```

Comment:
```suggestion
    :param bool eval_mode: set to ``False`` if one does not want to backpropagate through the spectral norm (default), set to ``True`` otherwise.
```